src/types/decoded-data.ts

```diff
@@ -2,11 +2,14 @@ export type DecodedDataRequest = {
   data: string
 }
 
+type ParamValue = string | string[]
```

Comment:
If you change this to reference itself, you can use it as `value` without needing arrays of arrays.

```suggestion
type ParamValue = (string | ParamValue)[]
```